pyproject.toml

```diff
@@ -29,6 +29,11 @@ dynamic = ["version"]
 [project.entry-points.hatch]
 nodejs = "hatch_nodejs_version.hooks"
 
+[project.urls]
+Homepage = "https://github.com/agoose77/hatch-nodejs-version"
+"Issue Tracker" = "https://github.com/agoose77/hatch-nodejs-version/issues"
+"Source Code" = "https://github.com/agoose77/hatch-nodejs-version"
```

Comment:
I took the name from the [pyproject-examples](https://github.com/repo-helper/pyproject-examples) repo. But you may prefer something more aligned with what this package is using:

```suggestion
Homepage = "https://github.com/agoose77/hatch-nodejs-version"
"Bug Tracker" = "https://github.com/agoose77/hatch-nodejs-version/issues"
"Repository" = "https://github.com/agoose77/hatch-nodejs-version"
```

Reply:
Related we could design to capitalize entries in https://github.com/agoose77/hatch-nodejs-version/blob/159d8960f101d2beccaf014948b4d489d6d09a66/hatch_nodejs_version/metadata_source.py#L157-L164

as that seems the common practice to align link in the side bar in pypi; e.g.

![image](https://user-images.githubusercontent.com/8435071/189601801-7fb1e3d6-6d02-40ee-9bd9-7f43ddaa30e7.png)

Reply:
Thanks for raising this!

RE the URL mapping, I think I'll just let the user override these labels if they need.